orderer/common/cluster/util.go

```diff
@@ -260,11 +260,11 @@ func VerifyBlockHash(indexInBuffer int, blockBuff []*common.Block) error {
 	if block.Header == nil {
 		return errors.New("missing block header")
 	}
-	if err := protoutil.VerifyTransactionsAreWellFormed(block); err != nil {
+	seq := block.Header.Number
+	dataHash, err := protoutil.BlockDataHash(block.Data)
+	if err != nil {
```

Comment:
As mentioned in the previous PR, this will crash when `block.Data==nil`, see comment below.